### ISSUE TITLE
Start-Transcript output is hard to parse

### ISSUE THREAD

**Issue by petemounce** [PowerShell/PowerShell-Docs#3593]
(This is feedback on the `Start-Transcript` cmdlet itself, the docs are great)

The output looks like

```
**********************
Windows PowerShell transcript start
Start time: 20190123080857
Username: BK-WINDOWS-V2-1\petermounce
RunAs User: BK-WINDOWS-V2-1\petermounce
Machine: BK-WINDOWS-V2-1 (Microsoft Windows NT 10.0.14393.0)
Host Application: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
Process ID: 7264
PSVersion: 5.1.14393.2636
PSEdition: Desktop
PSCompatibleVersions: 1.0, 2.0, 3.0, 4.0, 5.0, 5.1.14393.2636
BuildVersion: 10.0.14393.2636
CLRVersion: 4.0.30319.42000
WSManStackVersion: 3.0
PSRemotingProtocolVersion: 2.3
SerializationVersion: 1.1.0.1
**********************
Transcript started, output file is c:/blah\PowerShell_transcript.BK-WINDOWS-V2-1.0zNarV8d.20190123080857.txt
Logging a thing...
Done.
**********************
Command start time: 20190123080945
**********************
```
This is fine for human-reading, but very hard to parse for machines. For more context - I use Elastic's filebeat to hoover log files (and winlogbeat for event logs) into Elasticsearch. This enriches every log line with contextual information (like the things in the context-start section) so that _every log line_ gets turned into an event that is saved with that information. This makes my logs queryable in really useful ways within Kibana.

Would it be possible to do one of the following:
* Add a flag like `-OutputFormat=human|json` - human = as now, `json` = JSON blob per line (_not_ one single JSON object with an array of lines), where JSON blob contains the `message` from that line of transcript and also one field per item of contextual data
* Add a flag like `-LineTemplate` allowing the contextual information to be placed into the output according to how the user wants?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 61f8b565-4cc3-6a77-94e6-15b97ebcbca7
* Version Independent ID: afe03c1f-3dbf-62f7-722a-d845777cadbd
* Content: [Start-Transcript (Microsoft.PowerShell.Host)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.host/start-transcript?view=powershell-6)
* Content Source: [reference/6/Microsoft.PowerShell.Host/Start-Transcript.md](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Host/Start-Transcript.md)
* Product: **powershell**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

**Comment by sdwheeler** [comment]
@SteveL-MSFT Can you move this issue to the source repo?
